lib/versioning/composer/index.ts

```diff
@@ -14,13 +15,13 @@ export const urls = [
   'https://semver.mwl.be',
 ];
 export const supportsRanges = true;
-export const supportedRangeStrategies = [
+export const supportedRangeStrategies: RangeStrategy[] = [
   'bump',
-  'extend',
+  'widen',
   'pin',
   'replace',
   'update-lockfile',
-];
+] as RangeStrategy[];
```

Comment:
```suggestion
];
```

it seems to work without cast on ci